data/hooks/diagnosis/12-dnsrecords.py

```diff
@@ -1,13 +1,21 @@
 #!/usr/bin/env python
 
 import os
+import re
+
+from subprocess import CalledProcessError
+from datetime import datetime, timedelta
+from publicsuffix import PublicSuffixList
 
 from moulinette.utils.filesystem import read_file
+from moulinette.utils.process import check_output
 
 from yunohost.utils.network import dig
 from yunohost.diagnosis import Diagnoser
 from yunohost.domain import domain_list, _build_dns_conf, _get_maindomain
 
+PENDING_SUFFIX_LIST = ['ynh.fr', 'netlib.re']
```

Comment:
```suggestion
PENDING_SUFFIX_LIST = ['ynh.fr', 'netlib.re', 'nohost.me', 'noho.st']
```

Reply:
nohost.me and noho.st is already in public suffix list